pkgs/development/python-modules/async-tkinter-loop/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, pkgs
+, python3
+, buildPythonPackage
+, fetchPypi
+, poetry-core
+, pythonRelaxDepsHook
+}:
+buildPythonPackage rec {
+  pname = "async-tkinter-loop";
+  version = "0.8.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "async_tkinter_loop";
+    hash = "sha256-+9AvnYIZMWCbpCEKdbIadyU8zgyUlW/fRYYyDOxAzeg=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
+
+  buildInputs = [
+    poetry-core
+  ];
+
+  pythonRemoveDeps = [
+    "asyncio"
+  ];
+
+  meta = with lib; {
+    description = "Implementation of asynchronous mainloop for tkinter, the use of which allows using async handler functions";
```

```suggestion
    description = "Implementation of asynchronous mainloop for tkinter";
```
There is no need to change it, but it could be shortened.

---

I have been defaulting to using whatever the projects fill in on their github and similar. Should that not be preferable?

---

Basically, it is better to do so.
In this case, it was a bit long and redundant, IMHO.